taipy/gui/gui.py

```diff
@@ -962,8 +962,8 @@ def __upload_files(self):
                                 # remove file_path after it is merged
                                 part_file_path.unlink()
                     except EnvironmentError as ee:  # pragma: no cover
-                        _warn("Cannot group file after chunk upload", ee)
-                        return
+                        _warn(f"Cannot group file after chunk upload for {file.filename}", ee)
```

Comment:
Why not store the computed string in a reusable variable instead of duplicating the code?

Reply:
because this is an error and it should not happen, and we didn't do it before : bad reasons